helm/kontroler/values.yaml

```
@@ -66,7 +66,7 @@ certManager:
       selfSigned: {}
 
 kubeconfig:
-  secret: kubeconfig-secret
+  secret: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)